### PR TITLE
fix(issue#126): removed empty default actionButton in KtBanner

### PR DESCRIPTION
**Pull request by carsoli**
made `expandable` return boolean when no slot="expand" exists
if no actionText is provided and since it defaults to empty string, button doesn't show at all (v-if)
#126

### CODE REVIEW

File: packages/kotti-banner/src/Banner.vue [hunk 1]
```diff
@@ -65,7 +65,7 @@ export default {
 			}
 		},
 		expandable() {
-			return this.$slots.expand
+			return !!this.$slots.expand
```
Comment (AZeghers) [inline packages/kotti-banner/src/Banner.vue]: henlo pls Boolean(thing) ty

File: packages/kotti-banner/src/Banner.vue [hunk 2]
```diff
@@ -13,9 +13,9 @@
 			</div>
 			<div class="message" v-text="message" />
 			<div class="action" v-if="!expandable" @click="handleClick">
-				<button class="text" v-text="actionText" />
+				<button class="text" v-if="actionText.length" v-text="actionText" />
 			</div>
-			<div class="action" v-else @click="isExpand=!isExpand">
+			<div class="action" v-else @click="isExpand = !isExpand">
```
Comment (AZeghers) [inline packages/kotti-banner/src/Banner.vue]: outstanding linting
I wholeheartedly approve this change
Thanks for the catch

File: packages/kotti-banner/src/Banner.vue (hunk 1 above)
Comment (FlorianWendelborn) [reply packages/kotti-banner/src/Banner.vue]: Why is this code also in #125?

Comment (carsoli) [reply packages/kotti-banner/src/Banner.vue]: #124 u mean. idk what is #125 icet
and yes i just told u i reset head and made a separate issue (126)

will commit this here now.

**Pull request merged**